From bac7400034b48c1c0a38b0e3f8df52e7187defdb Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 27 Feb 2019 10:15:45 -0800
Subject: [PATCH 0521/1852] dt-bindings: display: xlnx: Add clock framework
 support to VTC bridge driver

commit 13ec9dd0d7a09d95d468aeb08c5fa5b8bd959be2 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds support for clock framework in the VTC bridge driver.

It breaks backward compatibility as it makes the clocks and clock-names
as required properties but compatible string isn't modified as IP
version hasn't changed.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,vtc.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vtc.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vtc.txt
index bafacea86b28..6a4d5bcc5e59 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vtc.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vtc.txt
@@ -18,10 +18,15 @@ Required properties:
  - compatible: value should be "xlnx,bridge-v-tc-6.1"
  - reg: base address and size of the VTC IP
  - xlnx,pixels-per-clock: Pixels per clock of the stream. Can be 1, 2 or 4.
+ - clocks: List of phandles for AXI Lite and Video Clock
+ - clock-names: Must contain "s_axi_aclk" and "clk" in same order as clocks listed
+   in clocks property.
 
 Example:
 	v_tc_0: v_tc@80030000 {
 			compatible = "xlnx,bridge-v-tc-6.1";
 			reg = <0x0 0x80030000 0x0 0x10000>;
 			xlnx,pixels-per-clock = <2>;
+			clock-names = "s_axi_aclk", "clk";
+			clocks = <&misc_clk_0>, <&misc_clk_1>;
 	};
-- 
2.31.1

